### PR TITLE
feat: add boost iostreams, multi_array, program_options, ptr_container

### PR DESCRIPTION
**Pull request by wep21**
- as title
- need skip url stability check

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (boost.ptr_container) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (boost.multi_array, boost.ptr_container) have been updated in this PR. Please review the changes.

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (boost.multi_array, boost.program_options, boost.ptr_container) have been updated in this PR. Please review the changes.

File: modules/boost.program_options/1.83.0/overlay/test/MODULE.bazel [hunk 1]
```diff
@@ -0,0 +1,7 @@
+bazel_dep(name = "boost.program_options")
+local_path_override(
```
Comment (fmeum) [inline modules/boost.program_options/1.83.0/overlay/test/MODULE.bazel:2]: This will be ignored, are you sure you want to check it in?

Comment (wep21) [reply modules/boost.program_options/1.83.0/overlay/test/MODULE.bazel:2]: You mean `local_path_override` will be skipped, right?

Comment (fmeum) [reply modules/boost.program_options/1.83.0/overlay/test/MODULE.bazel:2]: Yes

Comment (wep21) [reply modules/boost.program_options/1.83.0/overlay/test/MODULE.bazel:2]: I'm not sure why it is ignored, is there any way to fix it?

Comment (fmeum) [reply modules/boost.program_options/1.83.0/overlay/test/MODULE.bazel:2]: Overrides can only be applied by the root module. What do you want to achieve with this override?

Comment (wep21) [reply modules/boost.program_options/1.83.0/overlay/test/MODULE.bazel:2]: https://github.com/wep21/bazel-central-registry/blob/20feeda51c290d8d164954da8daf0055549bed43/modules/boost.program_options/1.83.0/presubmit.yml#L21
In the test phase, module path is pointed to `test` directory, so I've thought `test/MODULE.bazel` becomes the root module in the test phase. Is that wrong?

Comment (meteorcloudy) [reply modules/boost.program_options/1.83.0/overlay/test/MODULE.bazel:2]: This should actually work. Thanks for the explanation.

**Review by meteorcloudy — APPROVED** [review]

**Pull request merged**